src/common/i18n/translations/zh.ts

```diff
@@ -104,5 +104,263 @@ export const zh: Translations = {
+      title: "资料输入不完整",
+      body: "请选择至少一件物品才能完成交易。",
+      primaryActionText: "确定"
+    },
+    paymentCollected: {
+      title: "收到款项了吗？",
+      body: "此行动不能撤销。只有收到款项后才能继续。",
+      primaryActionText: "已收",
+      secondaryActionText: "未收"
+    },
+    outdatedAppRestart: {
+      title: "应用程序版本已过期",
+      body: "请重启应用程序，以下载最新版本。",
+      primaryActionText: "重启应用程序"
+    },
+    outdatedAppUpdate: {
+      title: "应用程序版本已过期",
+      body: "请通过[谷歌应用商店/苹果应用商店]更新应用程序。",
+      primaryActionText: "更新应用程序"
+    },
+    systemError: {
+      title: "系统错误",
+      body:
+        "连接目前出现问题。请重启应用程序。如果问题持续出现，请联系政府科技局。",
+      primaryActionText: "重启应用程序"
+    },
+    wrongFormatCountryCode: {
+      title: "格式错误",
+      body: "请输入有效的国家代码。",
+      primaryActionText: "确定"
+    },
+    wrongFormatCountryCodePhoneNumber: {
+      title: "格式错误",
+      body: "请输入有效的国家代码和联络号码。",
+      primaryActionText: "确定"
+    },
+    wrongFormatQRScanAgain: {
+      title: "格式错误",
+      body: "请再次扫描QR码或向您的负责人索取新QR码。",
+      primaryActionText: "确定"
+    },
+    wrongFormatValidCode: {
+      title: "格式错误",
+      body: "请输入有效的编号。",
+      primaryActionText: "确定"
+    },
+    wrongFormatDeviceCode: {
+      title: "格式错误",
+      body: "请再次扫描配备编号或向您的负责人索取新配备。",
+      primaryActionText: "确定"
+    },
+    wrongFormatID: {
+      title: "格式错误",
+      body: "请输入或扫描有效的证件号码。",
+      primaryActionText: "确定"
+    },
+    wrongFormatQRGetNew: {
+      title: "格式错误",
+      body: "请向您的负责人索取新QR码。",
+      primaryActionText: "确定"
+    },
+    alreadyUsedCode: {
+      title: "已使用",
+      body: "请输入独特的编号。",
+      primaryActionText: "确定"
+    },
+    alreadyUsedItem: {
+      title: "已使用",
+      body: "请扫描另一件没有与任何证件号码挂钩的物品。",
+      primaryActionText: "确定"
+    },
+    wrongFormatCode: {
+      title: "格式错误",
+      body: "请输入或扫描有效的编号。",
+      primaryActionText: "确定"
+    },
+    incompleteEntryCode: {
+      title: "资料输入不完整",
+      body: "请输入或扫描编号。",
+      primaryActionText: "确定"
+    },
+    wrongFormatNotValidDeviceCode: {
+      title: "格式错误",
+      body: "请扫描有效的配备编号。",
+      primaryActionText: "确定"
+    },
+    incompleteEntryScanDeviceCode: {
+      title: "资料输入不完整",
+      body: "请扫描配备编号。",
+      primaryActionText: "确定"
+    },
+    wrongFormatContactNumber: {
+      title: "格式错误",
+      body: "请输入有效的联络号码。",
+      primaryActionText: "确定"
+    },
+    insufficientQuota: {
+      title: "错误",
+      body: "限额不足",
+      primaryActionText: "确定"
+    },
+    invalidQuantity: {
+      title: "错误",
+      body: "数量无效",
+      primaryActionText: "确定"
+    },
+    categoryDoesNotExist: {
+      title: "错误",
+      body: "找不到物品类别",
+      primaryActionText: "确定"
+    },
+    systemErrorServerIssues: {
+      title: "系统错误",
+      body: "服务器目前出现问题。如果问题持续出现，请联系您的负责人。",
+      primaryActionText: "确定"
+    },
+    discardTransaction: {
+      title: "清除交易?",
+      body: "已扫描的物品将会被清除。",
+      primaryActionText: "取消",
+      secondaryActionText: "清除"
```

Comment:
On click on "Logout" in the sidebar I am getting this alert instead of "Confirm logout" alert. Can you check if we made a mistake?

Reply:
Apparently not giving enums results in them being assigned numbers, which caused collisions when performing the message to translation key mapping. I have since gave all enums string values to prevent such collisions!